master/getting-started/kubernetes/tutorials/using-calicoctl.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Using calicoctl in Kubernetes
+---
+
+There are two ways to run `calicoctl` in Kubernetes:
+
+- As a standalone binary
+- As a Kubernetes pod
+
+### a. Running calicoctl as a standalone Binary
+
+You can install calicoctl by [downloading the appropriate release]({{site.baseurl}}/{{page.version}}/releases) to any
+machine with access to your etcd cluster by setting `ETCD_ENDPOINTS`. For example:
+
+```
+ETCD_ENDPOINTS=http://etcd:2379 calicoctl get profile
+```
+
+>**Note for Kubeadm Deployments**
+>
+> Calico is not configured to use the etcd run by kubeadm on the Kubernetes master.
+> Instead, it launches its own instance of etcd as a pod, available at
+`http://10.96.232.136:6666`.
+>Ensure you are connecting to the correct etcd or you will not see any of the expected data.
+
+### b. Running calicoctl as a Kubernetes Pod
+
+The `calico/ctl` docker image can be deployed as a pod and used to run calicoctl
+commands. This pod will need to be configured for the Kubernetes environment it is in.
+
+>**Note**
+>
+>When calicoctl is run as a Pod, the `calicoctl node ...` suite of commands is not available.
+
+
+For **etcd backend** (e.g. kubeadm):
```

Comment:
This reads a bit awkwardly - it probably should be "For _the_ etcd backend" or "For deployments using the etcd backend"